### PR TITLE
trnka

### PR DESCRIPTION
**Pull request by petulkauwu**

### CODE REVIEW

File: .changes/fix.md
Comment (marekvospel) [inline .changes/fix.md:1]: Please use descriptive file names

File: .changes/fix.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+workshop-bot: minor
```
Comment (marekvospel) [inline .changes/fix.md:2]:
```suggestion
workshop-bot: patch
```

**Review by marekvospel — CHANGES_REQUESTED** [review]

File: .changes/fix.md
Comment (petulkauwu) [reply .changes/fix.md:1]: dost

**Review by marekvospel — APPROVED** [review]
LGTM! :beer:

**Pull request merged**